DokanNet/Dokan.cs

```diff
@@ -416,7 +33,7 @@ public static uint WaitForFileSystemClosed(this DokanInstance dokanInstance, uin
         /// <param name="driveLetter">Driver letter to unmount.</param>
         /// <returns><c>true</c> if device was unmount 
         /// -or- <c>false</c> in case of failure or device not found.</returns>
-        public static bool Unmount(char driveLetter)
+        public bool Unmount(char driveLetter)

```

Comment:
This and other functions under that are only calling native methods should be staying static no ? Having to create or get access to the single Dokan instance is not very convenient.

Reply:
It was deliberate, to enforce the usage of one Dokan instance that calls initialize and shutdown methods on constructor and dispose